test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatConfigTests.cs

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using NuGet.CommandLine.XPlat;
+using NuGet.Test.Utility;
+using Xunit;
+
+namespace NuGet.XPlat.FuncTest
+{
+    [Collection("NuGet XPlat Config Test Collection")]
```

Comment:
I'm not seeing a test fixture that corresponds to this collection. Does this do anything, or are you planning to use it in a future commit?

Reply:
I removed this for now since I'm not 100% sure if I will use it in the future!